### PR TITLE
document jmx new gc metric names option

### PR DESCRIPTION
**Pull request by arbll**
### What does this PR do?

A new option to use better java garbage collection metric names has been added to jmxfetch.

### CODE REVIEW

File: cmd/agent/dist/conf.d/jmx.d/conf.yaml.example [hunk 1]
```diff
@@ -29,6 +29,13 @@ instances:
   #   key_store_password: password
   #   rmi_registry_ssl: false # Optional, should be set to true if "com.sun.management.jmxremote.registry.ssl" is set to true on the target JVM.
   #
+  #   # Use better metric names for garbage collection metrics. The default value is false to ensure backward compatibility.
```
Comment (olivielpeau) [inline cmd/agent/dist/conf.d/jmx.d/conf.yaml.example]: just to be extra-clear, I'd write `Set to true to use better metric names [...]`

**Review by olivielpeau — DISMISSED** [review]
one nit, feel free to merge once addressed

**Comment by mdeggies** [comment]
Is this change reflected in the changelog? Our agent auto upgraded and all gc metrics were missing..

**Comment by arbll** [comment]
Hi @mdeggies,

You're right we should have advertised this in a more visible way. I'll add an entry to the agent changelog asap. 
Sorry for the inconvenience.

Arthur

**Pull request merged**